lib/octokit/client/licenses.rb

```diff
@@ -0,0 +1,62 @@
+module Octokit
+  class Client
+
+    # Methods for licenses API
+    #
+    module Licenses
+
+      LICENSES_PREVIEW_MEDIA_TYPE = "application/vnd.github.drax-preview+json".freeze
+
+      # List all licenses
+      #
+      # @see https://developer.github.com/v3/licenses/#list-all-licenses
+      # @return [Array<Sawyer::Resource>] A list of licenses
+      # @example
+      #   Octokit.licenses
+      def licenses(license_name = '', options = {})
+        options = ensure_deployments_api_media_type(options)
+        get "licenses", options
+      end
+
+      # List an individual license
+      #
+      # @see https://developer.github.com/v3/licenses/#get-an-individual-license
+      # @param license_name [String] The license name
+      # @return <Sawyer::Resource> An individual license
+      # @example
+      #   Octokit.license 'mit'
+      def license(license_name, options = {})
+        options = ensure_deployments_api_media_type(options)
+        get "licenses/#{license_name}", options
+      end
+
+      # Get a repository’s license
+      #
+      # @see https://developer.github.com/v3/licenses/#get-a-repositorys-license
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @return [Sawyer::Resource] Repository information including its license
+      # @example
+      #   Octokit.license_for_repository 'octokit/octokit.rb'
+      def license_for_repository(repo, options = {})
```

Comment:
I'm not sure we need this method. I know you were following the outline of the docs, but during the preview, I think we can get by with our existing method:

```ruby
client.repository("octokit/octokit.rb", :accept => "application/vnd.github.drax-preview+json").license.name
```

What do you think?